### DISCUSSION TITLE
Repopulate form with error bag

### DISCUSSION THREAD

**Discussion by shalahuddinalayubi** [laravel/framework#47707] [Ideas]
I am struggling with repopulate form with multiple form in single page.

I have multiple form in a single page. Each of them has its own error bag name.
1. form with foo bag name
2. form with bar bag name

Both has field named baz.

This for foo form: `<input type="text" name="baz" class="form-control"  value="{{ old('name', $foo->name) }}">`
This for bar form: `<input type="text" name="baz" class="form-control"  value="{{ old('name', $bar->name) }}">`

When the error validation occur in one of the field the both old value will be the same.

How can I differentiate for foo and bar to repopulate form with the same field named baz?

**Comment by yuan8** [comment]
if you have same name field. form will get last one for send to server.
seed you code. bar name is collected for value baz.
try debug your server use dd($request->all())

sugestion
try to change name.
or if you want collect foo and bar name try use array name. like name="baz[]"
